Topic 1/src/com/globant/topic1/exercise1/model/Email.java

```diff
@@ -0,0 +1,56 @@
+package com.globant.topic1.exercise1.model;
+
+public class Email {
+	// Final Fields
+	private static final String TO_EMAIL = "manager@email.com";
+	private static final String FROM_EMAIL = "notifyEmailService@email.com";
+
+	// Private instance Field
+	private String subject;
+
+	// Constructor
+	public Email(String subject) {
+		this.subject = subject;
```

Comment:
In this abstraction we need to create an email and at this time, we need to put the body and from/to .